Formula/kubefwd.rb

```diff
@@ -0,0 +1,17 @@
+class Kubefwd < Formula
+    desc "Bulk port forwarding Kubernetes services for local development"
+    homepage "https://github.com/txn2/kubefwd"
+    url "https://github.com/txn2/kubefwd.git"
+    version "1.9.2"
+    sha256 "3f49349f7de2c8f60ae4d1248be4f936c9082d2b23cf9fa093caef20a60685d4"
+  
+    depends_on "go@1.11" => :build
```

Comment:
If it doesn't build with the latest version it might not be a good fit for homebrew-core.